### ISSUE TITLE
🌐  i18n notification messages

### ISSUE THREAD

**Issue by tjarbo** [tjarbo/discord-moodle-bot#127]
> **As a** User
> **I want to** set the language of the notification messages
> **because** I cannot speak German 
 

## Description:
Right now, we are using many (two) different languages at backend. All ApiErrors are in English while the messages for moodle are in German. This user story is focused on i18n all notification messages. The language of the backend can be selected by an environment variable with the same validators like "NODE_ENV" . The default language should be english.

This user story is blocked by #108!
This user story is connected to #126 - make sure to stay in sync with the implementations of this issue

### 🟢 In scope:
* Create new ENV var and set default value to "en"
* Extend all message templates with the English language 
* The language translation will be part of the Message class
* 

### 🔴 Not in scope:
* Translate the api responses

## What should be the result?
GIVEN
WHEN I receive a Moodle notification
THEN the notification should be in English

GIVEN language='de'
WHEN I receive a Moodle notification
THEN the notification should be in German
